apiclient/types/mcpserver.go

```diff
@@ -48,7 +49,7 @@ type MCPServer struct {
 	Configured             bool     `json:"configured"`
 	MissingRequiredEnvVars []string `json:"missingRequiredEnvVars,omitempty"`
 	MissingRequiredHeaders []string `json:"missingRequiredHeader,omitempty"`
-	CatalogID              string   `json:"catalogID"`
+	CatalogEntryID         string   `json:"catalogID"`
```

Comment:
Might be worthwhile changing the JSON tag on this. This isn't a catalog ID; it's a catalog entry ID.

Reply:
Yeah, change it. There is probably some UI code that would also need to change.